python/ray/_private/parameter.py

```diff
@@ -12,6 +12,10 @@ class RayParams:
     """A class used to store the parameters used by Ray.
 
     Attributes:
+        external_addresses (str): The address of external Redis server to
```

Comment:
Better name it something like "external_gcs_storage_addresses". 
And we should document that this is addresses of the GCS's backend of storage service. Currently the storage is Redis, and we may have other options in the future.

Reply:
"external_gcs_storage_addresses" is more explicit than just `external_addresses`, and I like it.
@rkooo567 What do you think?

Reply:
I think we don't want to have gcs here because it is the implementation detail (that was the primary reason why we used `address` instead of `gcs_address`). Btw, we are planning to remove Redis in the next quarter, and we should probably hold off this change. cc @ericl

Reply:
@rkooo567 Maybe you mixed up `gcs` with `redis`. Sure we DO NOT wanna include `redis` suffix/prefix in naming as implementation would drift in the future, but `gcs` is a SURE name in Ray. :-p

Reply:
No. Gcs was the candidate for the api and it was rejected for the same reason

Reply:
Ohhh, didn't know about the gcs thing. Maybe we can change the name to "external_backend_storage_addresses" or just "external_storage_addresses"?

Reply:
or "external_metadata_storage_addresses"? Adding "metadata" to distinguish it from object store.